tests/bam/bam_pb.robot

```diff
@@ -515,7 +515,7 @@ BEPB_DIMENSION_BA_BV_RELATION_EVENT
 	Kindly Stop Broker  True
 
 BEPB_DIMENSION_TIMEPERIOD
-	[Documentation]	use of pb_dimension_timeperiod message.
+	[Documentation]	use of pb_dimension_timeperiod Adding BAM and Lua configurations to the broker,connects to  database and executes an SQL insert statement, checking whether the event message is correctly logged by the system.
```

Comment:
connects to database looks too much since it is mandatory to execute the sql insert

Reply:
I think you can make something lighter here and even more explicit.
In fact, we add bam outputs to broker (needed because of the dimension timeperiod that is specific to bam).
But the test inserts some timeperiod in the configuration database and checks if this configuration is well received by broker. The lua configuration is just a tool for us to check the event in broker, not a goal.
I give you ideas, but your words are better than mine.